client/src/controllers/SwapController.ts

```diff
@@ -35,9 +35,21 @@ import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
  *    data-w-swap-target-value="#listing-results"
  *  />
  *
+ * @example - A single button that will update the results
+ *  <div id="results"></div>
+ *  <button
+ *    id="clear"
+ *    data-controller="w-swap"
+ *    data-action="input->w-swap#replaceLazy"
+ *    data-w-swap-src-value="path/to/results/?type=bar"
+ *    data-w-swap-target-value="#results"
+ *  >
+ *    Clear owner filter
+ *  </button>
+ *
  */
 export class SwapController extends Controller<
-  HTMLFormElement | HTMLInputElement
+  HTMLFormElement | HTMLInputElement | HTMLButtonElement
```

Comment:
I updated the TypeScript spec and example docstring to allow `HTMLButtonElement` because we technically already use this for clearing the active filters:

https://github.com/wagtail/wagtail/blob/e06ec6d28eb2044e68ebcda6bc5b2c0e6736b9f8/wagtail/admin/templates/wagtailadmin/shared/active_filters.html#L9-L11

Note that this only affects TypeScript so it shouldn't change anything at runtime. That said, I added a test anyway, to ensure the feature and the fix work even when the controller is attached to an element that's not associated with a form.